New Jobs/War_Correspondent.txt

```diff
@@ -0,0 +1,26 @@
+## Abstract
+
+War Correspondent is a journalist role, meant to be a sort've independent/marine aligned CL. It's meant to provide more roleplay opportunities and add more flavour to the game I guess.
+
+## Goals
+
+1. Create a role that's interesting and provides new oppirtunities along with spicing up the game I suppose
+2. Create a roleplay-centric role that's differentiated from the CL enough to be its own thing
+
+## Non-goals
+
+1. Making another CL
+2. Shifting the balance of the game
+3. Giving new opportunities for grief
+
+## Content
+
+(To be added) When WC players spawn in they get a choice from either Marine or Independent which sets them on their alignment for the round, they get a whole new office, vendor, and new clothes. A new fancy camera also spawns in there that can take more photos then other cameras. Also with this MR I've changed camera code a bit to include the amount of photos left whenever you examine a camera.
```

Comment:
This is implementation detail. You should be describing the design of your feature, not how it is to be implemented.